README-HE.md

```diff
@@ -0,0 +1,192 @@
+- [למה קוד פתוח ואיך?](https://careerkarma.com/blog/open-source-projects-for-beginners/) ע״י [james-gallagher@](https://careerkarma.com/blog/author/jamesgallagher/)
+- [כיצד להתחיל עם קוד פתוח](https://www.hackerearth.com/getstarted-opensource/) ע״י Sayan Chowdhury
+- [לאיזה קוד פתוח כדי לתרום](https://kentcdodds.com/blog/what-open-source-project-should-i-contribute-to/) ע״י Kent C. Dodds
+
+</p>
+</div>
+<div dir="rtl" style="direction:rtl;text-align:right;">
+<p>
+
+## שימוש ב-Version Control
+- [Think Like (a) Git](http://think-like-a-git.net/) - מדריך גיט ל״מתחילים מתקדמים״ מתקשים, המחפשים אסטרטגיה פשוטה להתנסות עם גיט.
+- [Try Git](https://try.github.io/) - למדו גיט ב-15 דקות ישירות מהדפדפן בחינם.
+- [Everyday Git](https://git-scm.com/docs/giteveryday) - סט מינימלי שימושי של פקודות לשימוש גיט יומיומי
+- [!Oh shit, git](https://ohshitgit.com/) - כיצד לצאת מטעויות גיט נפוצות, כתוב באנגלית; ראו גם [!Dangit, git](https://dangitgit.com/) עבור עמוד ללא קללות.
+- [מדריכי גיט של Atlassian](https://www.atlassian.com/git/tutorials) - מגוון מדריכים על השימוש בגיט.
+- [שליף לגיטהאב גיט](https://education.github.com/git-cheat-sheet-education.pdf) (PDF)
+- [הויקיפדיה של freeCodeCamp על מקורות גיט](https://forum.freecodecamp.org/t/wiki-git-resources/13136)
+- [GitHub Flow](https://www.youtube.com/watch?v=juLIxo42A_s) - שיחת גיטהאב על כיצד ליצור pull request. מתחיל ב-(42:06)
+- [מקורות הלימוד של גיטהאב](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/git-and-github-learning-resources) - מקורות למידה עבור גיט וגיטהאב.
+- [Pro Git](https://git-scm.com/book/en/v2) - הספר Pro Git בשלמותו, נכתב על ידי Scott Chacon ו-Ben Straub ופורסם על ידי Apress.
+- [Git-it](https://github.com/jlord/git-it-electron) - מדריך גיט לאפליקצית הדסקטופ.
+- [Flight Rules לגיט](https://github.com/k88hudson/git-flight-rules) - מדריך על מה לעשות כשדברים משתבשים.
+- [מדריך גיט בספרדית למתחילים](https://platzi.github.io/git-slides/#/) - מדריך שלם של שקופיות על גיט וגיטהאב בספרדית. Una guía completa de diapositivas sobre git y GitHub explicadas en Español.
+- [Git Kraken](https://www.gitkraken.com/git-client) - אפליקצית דסקטופ לגיט, ויז׳ואלית, אינטרקטיבית ו-cross-platform.
+- [טיפים לגיט](https://github.com/git-tips/tips) - אוסף טיפים וטריקים נפוצים בגיט.
+- [שיטות עבודה מומלצות בגיט](https://sethrobertson.github.io/GitBestPractices/) - בצעו commit לעיתים קרובות, שפרו בהמשך, פרסמו פעם אחת: שיטות העבודה המומלצות בגיט.
+- [מדריך גיט אינטרקטיבי](https://learngitbranching.js.org/) - למדו גיט בדרך הויז׳יאלית והאינטרקטיבית ביותר.
+
+</p>
+</div>
+<div dir="rtl" style="direction:rtl;text-align:right;">
+<p>
+
+## ספרים בנושא קוד פתוח
+- [יצירת תוכנה בקוד פתוח](https://producingoss.com/) - זהו ספר על הצד האנושי של פיתוח קוד פתוח. הספר מתאר כיצד פרויקטים מוצלחים עובדים, ציפיות האנשים והמפתחים ותרבות תכונה חינמית.
```

Comment:
Believe that it should be `תוכנה` and not `תכונה`

Reply:
fixed!